documentation/USAGE.rst

```diff
@@ -558,108 +558,118 @@ can often be substituted for the flux in an equation
 
```

Comment:
The condition `on $f=f_0$` is first used at the top of this section, but neither term is ever defined.

Reply:
I completely changed  how the boundary faces are identified, both to make clearer and  more rigorous (I think)